test/modules/sdk/objections/service.spec.unit.ts

```diff
@@ -63,4 +63,16 @@ describe("objections SDK service unit tests", () => {
 
     expect(mockAxiosRequest).toBeCalled();
   });
+
+  it("objections API is called when posting an attachment", () => {
+
+    objectionsSdk.addAttachment("companyNumber",
+      "token",
+      "objectionId",
+      new Buffer("Buffer"),
+      "fileName",
+    );
+
+    expect(mockAxiosRequest).toBeCalled();
```

Comment:
could you capture the argument (axiosConfig) passed to mockAxiosRequest and check that the file info has been set correctly? maybe check the buffers and filenames match
see https://github.com/companieshouse/extensions-web/blob/master/src/test/client/apiclient.add.attachment.spec.unit.ts 
and https://jestjs.io/docs/en/mock-functions#mock-property for example on capture args

Reply:
or .calledWith(stuff) might work